patches/chrome-browser-ui-android-toolbar-java-src-org-chromium-chrome-browser-toolbar-top-ToolbarPhone.java.patch

```diff
@@ -18,17 +18,23 @@ index 1e7a5d49490b23273e388d5dce2f525877873272..d468e6f96c3c94e33c029d0c02e8d128
      }
  
      @Override
-@@ -699,7 +701,8 @@ public class ToolbarPhone extends ToolbarLayout implements OnClickListener, TabC
+@@ -699,6 +701,7 @@ public class ToolbarPhone extends ToolbarLayout implements OnClickListener, TabC
       * @return The right bounds of the location bar after accounting for any visible right buttons.
       */
      private int getBoundsAfterAccountingForRightButtons() {
--        if (mStartSurfaceScrollFraction == 1.0f) return mToolbarSidePadding;
-+      if (true) return Math.max(mToolbarSidePadding, BraveToolbarLayout.class.cast(this).getBoundsAfterAccountingForRightButtonsImpl(mToolbarButtonsContainer));
-+      if (mStartSurfaceScrollFraction == 1.0f) return mToolbarSidePadding;
++        if (true) return Math.max(mToolbarSidePadding, BraveToolbarLayout.class.cast(this).getBoundsAfterAccountingForRightButtonsImpl(mToolbarButtonsContainer));
```

Comment:
This one not related to the issue, but just fixes unnecessary change is the patch.